getting-started.md

```diff
@@ -5,156 +5,93 @@ nav_title: Getting Started
 permalink: /getting-started/
 ---
 
-Flutter currently supports developers on Mac and Linux (64-bit).
-Windows support is planned but currently incomplete.
+This quickstart guide describes how to create and run your first Flutter app on iOS or Android.
 
-## Dependencies
+[Before you begin](#before_you_begin)<br>
```

Comment:
This is not really what `<br>` is supposed to be used for. Maybe a bulletted list would be better?